sequencer/src/persistence/sql.rs

```diff
@@ -1364,15 +1405,6 @@ impl SequencerPersistence for Persistence {
 
             offset += batch_size;
 
-            tx.upsert(
```

Comment:
these changes should be in both of the PR's. I don't know why they are being removed